<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="SideMenuWidget" name="sidemenu_widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>201</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="pages_widget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>691</width>
     <height>531</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="HomePageWidget" name="home_page_widget"/>
   <widget class="ScanPageWidget" name="scan_page_widget"/>
   <widget class="UpdatePageWidget" name="update_page_widget"/>
   <widget class="ThreatPageWidget" name="threat_page_widget"/>
   <widget class="SettingPageWidget" name="setting_page_widget"/>
   <widget class="AboutPageWidget" name="about_page_widget"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="refresh_pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe MDL2 Assets</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string></string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mini_pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe MDL2 Assets</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string></string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="close_pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe MDL2 Assets</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string></string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>891</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>DEFENDER PRO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="background_image_label">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>0</y>
     <width>911</width>
     <height>601</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="background_mask_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>711</width>
     <height>601</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background_image_label</zorder>
  <zorder>background_mask_label</zorder>
  <zorder>title_label</zorder>
  <zorder>sidemenu_widget</zorder>
  <zorder>pages_widget</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HomePageWidget</class>
   <extends>QWidget</extends>
   <header>homepagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SideMenuWidget</class>
   <extends>QWidget</extends>
   <header>sidemenuwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScanPageWidget</class>
   <extends>QWidget</extends>
   <header>scanpagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UpdatePageWidget</class>
   <extends>QWidget</extends>
   <header>updatepagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ThreatPageWidget</class>
   <extends>QWidget</extends>
   <header>threatpagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SettingPageWidget</class>
   <extends>QWidget</extends>
   <header>settingpagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AboutPageWidget</class>
   <extends>QWidget</extends>
   <header>aboutpagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
